llvm/unittests/ADT/MapVectorTest.cpp

```diff
@@ -267,6 +267,30 @@ TEST(MapVectorTest, NonCopyable) {
   ASSERT_EQ(*MV.find(2)->second, 2);
 }
 
+TEST(MapVectorTest, EnumClassKey) {
```

Comment:
Why test this through MapVector rather than DenseMap directly?

Reply:
There's no DenseMapTest.cpp. Perhaps just use DenseMap directly in MapVectorTest.cpp?

Reply:
I see a DenseMapTest.cpp here: https://github.com/llvm/llvm-project/blob/main/llvm/unittests/ADT/DenseMapTest.cpp